### PR TITLE
{bio}[intel/2017a] Biopython v1.70

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2573.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/77d6aa2290f653af7b822471f7643b66 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2070.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/46ccdfc716db803f23d7acd0ff5c8812 for a full test report.

File: easybuild/easyconfigs/b/Biopython/Biopython-1.70-intel-2017a-Python-3.6.1.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+homepage = 'http://www.biopython.org'
+description = """Biopython is a set of freely available tools for biological computation written
+in Python by an international team of developers. It is a distributed collaborative effort to
+develop Python libraries and applications which address the needs of current and future work in
+bioinformatics. """
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://biopython.org/DIST']
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (verdurin) [inline easybuild/easyconfigs/b/Biopython/Biopython-1.70-intel-2017a-Python-3.6.1.eb]: Checksum here?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2070.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9fb1b2d5b2776bc89e241549745ae665 for a full test report.

**Comment by verdurin** [comment]
@boegel my build completed in the end. Looks fine.

**Comment by verdurin** [comment]
Merging, thanks @boegel

**Pull request merged**